CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+Fix: Do not intercept error about DEVICE_NOT_FOUND in findOrCreate device (iotagent-node-lib#889)
```

Comment:
```suggestion
Fix: Do not intercept error about DEVICE_NOT_FOUND in findOrCreate device (iotagent-node-lib#889)
```

Similar fix needed in IOTA-UL code? Or it is alredy fixed there?

Reply:
No, only this agent re implement findOrCreate function of iotagent-node-lib

Reply:
NTC

Reply:
twin PR https://github.com/telefonicaid/iotagent-ul/pull/439